### DISCUSSION TITLE
Améliorer le tableau Excel de suivi des achats

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1042] [Idées]
2 suggestions d'une utilisatrice (S. T.) le 08/02 :
- Pourquoi distinguer entre le local et le non-local sur le tableau de suivi des achats? Cette distinction complexifie inutilement la chose.
- La colonne "Quantité" n'est pas utile à ses yeux.

**Comment by gauthierandre** [comment]
Pour la colonne local, je ne sais pas si tu l'as en tête, celle-ci répond au besoin de beaucoup de collectivités d'avoir un suivi des approvisionnements locaux, sans multiplier les outils. Ce n'est pas dans la loi EGAlim mais cela permet d'avoir un seul outil pour suivre les achats. La valorisation du local est un enjeu très important pour les élu.e.s des collectivités territoriales.
Cela dit, le tableur étant en local, l'utilisateur a la liberté la plus totale pour modifier le tableur et le faire coller à son usage :)
